package.json

```diff
@@ -162,6 +160,12 @@
                 "when": "editorTextFocus && editorLangId == lean"
             },
             {
+                "command": "lean.displayList",
+                "key": "ctrl+shift+alt+enter",
```

Comment:
`ctrl+shift+enter` is already emacs :-) I generally like to use `alt`-versions, it feels natural. And in this case most keyboard have `ctrl+shift+alt` easily accessible with one hand.

Another idea would be to go full emacs and use a prefix like `ctrl+L` and then have `ctrl+L g`, `ctrl+L l`, `ctrl+L s` or similar for Lean interactions.